tests/Fixtures/Integration/priority/phpdoc_to_return_type,no_superfluous_phpdoc_tags.test

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Integration of fixers: phpdoc_to_return_type,no_superfluous_phpdoc_tags.
+--RULESET--
+{"phpdoc_to_return_type": true, "no_superfluous_phpdoc_tags": true}
+--REQUIREMENTS--
+{"php": 70000}
+--EXPECT--
+<?php
+
+class Test {
+    /**
+     */
+    public function foo(): int {
+    }
+}
+
+--INPUT--
+<?php
+
+class Test {
+    /**
+     * @return int
+     */
+    public function foo() {
+    }
+}
```

Comment:
maybe best to update to
```
/**
 * @return int
 */
public function foo() {
    return 1;
}
```

so when the fixer get updates to check if some is actually returned this test still works

Reply:
Updated.